packages/verifiable-credentials/src/credentials.ts

```diff
@@ -127,10 +142,15 @@ export default class Credentials {
 	 * @param {string} didRegistryEndpoint
 	 * @param {string} currentDateTime to allow the client to migrate cases where the datetime is incorrect on the local computer
 	 */
-	public async verifyCredential(vcJwt: string, resolverConfig?: Web3ResolverConfigurationOptions, currentDateTime?: string): Promise<any> {
+	public async verifyCredential(
+		vcJwt: string,
+		options: VerifyCredentialOptions = {removeOriginalFields: false},
+		resolverConfig?: Web3ResolverConfigurationOptions,
+		currentDateTime?: string,
```

Comment:
`options` should be last for backward compatibility

Reply:
Done